### ISSUE TITLE
Add nice way of printing datasets, tasks etc

### ISSUE THREAD

**Issue by mfeurer** [openml/python#38]

**Comment by mfeurer** [comment]
@joaquinvanschoren you used `pprint` in one of your examples. I would prefer if the `__str__` method of the respective classes provide a nice output. What do you think?

**Comment by joaquinvanschoren** [comment]
Yes, I also prefer to implement a good __str__ method, but they were not
available at the time.

**Comment by mfeurer** [comment]
Okay, I'll have a look at this.
